test/api.sh

```diff
@@ -28,9 +28,17 @@ uuid_str()
     LC_ALL=C; cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
 
+_shakedown() {
+  shakedown "$@" -H "Authorization: Bearer ${FLIPT_TOKEN:-""}"
+}
+
+_curl() {
+  curl "$@" -H "Authorization: Bearer ${FLIPT_TOKEN:-""}"
+}
+
 step_1_test_health()
 {
-    shakedown GET "/health"
+    _shakedown GET "/health"
```

Comment:
nit, this doesnt / shouldnt require auth, or should it? 🤔

Reply:
Ahh yes, so it doesn't require auth. This line can go and should go.
(wonders if I should rename method `authedShakedown`).

Health and Ready should be unauthenticated. But perhaps meta should be authed.